workflows/prognostic_c48_run/runtime/steppers/machine_learning.py

```diff
@@ -310,6 +310,8 @@ def __call__(self, time, state):
 
     def get_diagnostics(self, state, tendency):
         diags = compute_diagnostics(state, tendency, self.label, self.hydrostatic)
+        momentum_diags = compute_ml_momentum_diagnostics(state, tendency)
+        diags.update(momentum_diags)
         return diags, diags[f"net_moistening_due_to_{self.label}"]
 
     def get_momentum_diagnostics(self, state, tendency):
```

Comment:
We should be able to delete this method on all steppers now (which is great!!)

Reply:
Indeed, thanks for pointing that out!